spec/models/fee_calculator_spec.rb

```diff
@@ -146,5 +146,24 @@
         it { is_expected.to eq(400) }
       end
     end
+    context '土日は値段が違う' do
+      let(:attr) do
+        {
+         'type' => 'wday',
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.